brainstormforce/ultimate-addons-for-gutenberg
# Fix: Masonry Feature - Broken with new core/gallery.

### Description

- Task - https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/f8588fc4-5f3a-4254-bab2-575c9fd9b467?view=6d3bcc3d-b047-4739-8e05-d9621e9f9445&t=attributes


### Screenshots


### Types of changes




- Breaking change 

### How has this been tested?

- Backword - creates masonry using core/gallery in master and then switch to `next-release`.
- Install Gutenberg plugin.
- Drag and drop core/gallery add images
- Check Masonry tab is visible or not.
- Switch to `masonry-feature-broken-with-new-core-gallery`  and check by enabling the masonry feature.

### Checklist:
